lib/src/samples/authenticate_with_oauth.dart

```diff
@@ -0,0 +1,24 @@
+import 'package:flutter/material.dart';
```

Comment:
Need to add the apache licenses for both sample files

Reply:
added